content/zh-cn/docs/reference/glossary/event.md

```diff
@@ -10,7 +10,6 @@ tags:
 - core-object
 - fundamental
 ---
-
 <!--
```

Comment:
We don't have strong preference over whether the newline is needed or not.

For instance, [the zh localization guide](https://kubernetes.io/zh-cn/docs/contribute/localization_zh/#frontmatter-%E7%9A%84%E5%A4%84%E7%90%86) have a newline between.